docs/schema.md

```diff
@@ -388,6 +388,7 @@ The defined ecosystems are:
 | `Android`  | The Android ecosystem; the `name` field is the Android component name that the patch applies to, as shown in the [Android Security Bulletins](https://source.android.com/security/bulletin) such as `Framework`, `Media Framework` and `Kernel Component`. The exhaustive list of components can be found at the [Appendix](#android-ecosystem-components). |
 | `GitHub Actions` | The GitHub Actions ecosystem; the `name` field is the action's repository name with owner e.g. `{owner}/{repo}`. |
 | `Pub` | The package manager for the Dart ecosystem; the `name` field is a Dart package name. |
+| `conan.io` | The conan.io ecosystem for C and C++; the `name` field is a conan package name.  |
```

Comment:
We typically stylize ecosystems the way it's typically referred to in canonical documentation. 

E.g. "crates.io" per https://www.rust-lang.org/governance/teams/crates-io. PyPI per https://pypi.org.

Looking at https://conan.io, should this be "Conan", or perhaps "ConanCenter" instead?

Reply:
conan documentation doesn't mention ecosystem at all.
google has more results for `conan ecosystem` than `conan.io ecosystem` or `conancenter ecosystem`. so I suppose `conan ecosystem` is canonical one.
`conan.io` is just FQDN. `ConanCenter` is a place where packages built and stored. it's a part of the ecosystem for sure, but it won't work without other things (like conan client).
@prince-chrismc lemme know if you agree.

Reply:
Thanks! We generally respect capitalization, so if we go with "Conan" the C should be capitalized. 

Can you explain a bit more about why `ConanCenter` doesn't work? The intention of these ecosystems is to unambiguously identify a package that exists in some open repository. If there are multiple repositories, then `Conan` on its own could be ambiguous because it's not clear to which repository a package belongs.

Reply:
conan is distributed by design and provides multiple repositories (remotes). e.g. everyone can easily run their own conan server. ConanCenter is just one of them.
I think it makes sense if we want to target specifically ConanCenter packages, to avoid ambiguity.
so I am changing to capiital `ConanCenter` then.

Reply:
Thanks! For the purposes of OSV, referring specifically to the main Conan repo for open source packages makes the most sense.

Do we want to wait for @prince-chrismc's input as well?

Reply:
no, I think we all have an agreement already